### PR TITLE
Populate current date as that of the appointment

### PR DESCRIPTION
**Pull request by jongilbraith**

### CODE REVIEW

File: app/controllers/appointment_summaries_controller.rb [hunk 1]
```diff
@@ -96,6 +96,7 @@ def appointment_summary_params
       .fetch(:appointment_summary, {})
       .permit(AppointmentSummary.editable_column_names)
       .merge(user: current_user, telephone_appointment: telephone_appointment?)
+      .reverse_merge(date_of_appointment: Time.zone.now.to_date)
```
Comment (benlovell) [inline app/controllers/appointment_summaries_controller.rb]: `Date.current` is a little more concise (and entirely equivalent)

Comment (jongilbraith) [inline app/controllers/appointment_summaries_controller.rb]: @benlovell amended

**Comment by jongilbraith** [comment]
[As a f2f guider when generating a summary document I am given suggestions of valid email address formats when entering invalid emails addresses](https://trello.com/c/4hVlQkgC/199-as-a-f2f-guider-when-generating-a-summary-document-i-am-given-suggestions-of-valid-email-address-formats-when-entering-invalid-e)

**Comment by jongilbraith** [comment]
[As a f2f guider when I generate a summary document the 'Date of appointment' is set to the date of when I access the summary document generator by default.](https://trello.com/c/TmL2nmKu/198-as-a-f2f-guider-when-i-generate-a-summary-document-the-date-of-appointment-is-set-to-the-date-of-when-i-access-the-summary-docum)

**Pull request merged**